test/quantization/fx/test_quantize_fx.py

```diff
@@ -4545,14 +4546,21 @@ def forward(self, inputs: torch.Tensor, h0: torch.Tensor, c0: torch.Tensor):
                 x = self.my_lstm(inputs, (h0, c0))
                 return x
 
-        class UserLSTM(torch.ao.nn.quantizable.LSTM):
+        # Construct a BackendConfig that supports qint32 for certain ops
+        qint32_dtype_config = DTypeConfig(input_dtype=torch.qint32, output_dtype=torch.qint32)
+        my_backend_config = get_qnnpack_backend_config()
```

Comment:
optional nit: might be cleaner to build from scratch.  `get_qnnpack_backend_config` can change, and it's not clear which parts of it are needed to test this PR

Reply:
Actually I tried this and it ended up inflating the test code a lot, so I prefer to leave this as is. Also this is likely how our customers will end up using our APIs, so having this here will serve as a practical example

Reply:
> Also this is likely how our customers will end up using our APIs, so having this here will serve as a practical example

this is unsafe though, I would not recommend our users to do that, I would usually request changes if I see something like that in a production path.  For example, if their backend doesn't support some other op that qnnpack supports, then they will get silently incorrect results.